pkg/scheduler/metrics/metrics.go

```diff
@@ -67,6 +75,18 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	PreemptionVictims = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Subsystem: schedulerSubsystem,
+			Name:      "pod_preemption_victims",
+			Help:      "Total number of victims found",
```

Comment:
nit: s/"Total number of victims found"/"Number of selected preemption victims"/

Reply:
Done.